src/components/product/ProductCard/product-card.scss

```diff
@@ -5,22 +5,25 @@
   min-width: 14rem;
   height: 100%;
   padding: var(--space-1) var(--space-1) var(--space-2);
+  border: 1px solid transparent;
   border-radius: var(--border-radius-default);
-  transition: box-shadow .5s ease;
+  transition: box-shadow .5s ease, border .5s ease;
 
   &:hover,
   &:focus {
     background-color: var(--bg-neutral-lightest);
   }
 
-  &:hover {
-    outline: 1px solid var(--color-border-display);
-    box-shadow: var(--box-shadow);
+  @media (hover: hover) {
+    &:hover {
+      border: 1px solid var(--color-border-display);
+      box-shadow: var(--box-shadow);
+    }
   }
 
   &:focus,
   &:focus-within {
-    outline: 2px solid var(--bg-focus-ring);
+    box-shadow: 0 0 0 1px var(--color-neutral-0), 0 0 0 var(--border-width-3) var(--bg-focus-ring);
```

Comment:
We are also using this on `sku-selector.scss`. Do you think we will need to apply this also to other components? If so, we could extract it to a mixin in the future.

Reply:
Actually. I just did it [here](https://github.com/vtex-sites/base.store/pull/219/commits/5ba320546e1a1dea64e8287b43bb1ae38b3dd724). Lemme know if you have any thoughts about it.